assets/app/views/directives/osc-key-values.html

```diff
@@ -23,7 +25,15 @@
                  spellcheck="false"
                  osc-input-validator="value">
         </div>
-        <button class="btn btn-default" ng-click="addEntry()" ng-disabled="form.key.$error.oscKeyValid || form.value.$error.oscValueValid">Add</button>
+        <button
+          class="btn btn-default"
+          ng-click="addEntry()"
+          ng-disabled="form.$invalid || !key || !value">
```

Comment:
Disabled when invalid || key || value missing.  Seems to cover our bases (unless there is somewhere we would want to allow a key w/o a value?)

@spadgett